nionswift_plugin/usim/RonchigramCameraSimulator.py

```diff
@@ -343,6 +346,16 @@ def get_frame_data(self, readout_area: Geometry.IntRect, binning_shape: Geometry
             self._needs_recalculation = True
         if self.instrument.sample != self.__last_sample:
             self._needs_recalculation = True
+        scan_device: typing.Optional[ScanDevice.Device] = Registry.get_component("scan_device")
+        probe_position: typing.Optional[Geometry.FloatPoint] = Geometry.FloatPoint(0.5, 0.5)
+        if scan_device:
```

Comment:
Here too... a comment to explain this section?

Also: this section seems to be almost the same as the one in the EELS camera simulator. Does it make sense to make a common method?